homeassistant/components/bmw_connected_drive.py

```diff
@@ -16,7 +16,7 @@
     CONF_USERNAME, CONF_PASSWORD
 )
 
-REQUIREMENTS = ['bimmer_connected==0.3.0']
+REQUIREMENTS = ['bimmer_connected==0.4.0']
```

Comment:
Please update to version `0.4.1`. I fixed some exceptions there and it's currently already in use on master

Reply:
Done